tensorflow/python/eager/def_function_test.py

```diff
@@ -938,6 +938,36 @@ def f(x):
 
     self.assertLen(logs.output, 1)
     self.assertIn('Tracing is expensive', logs.output[0])
+  
+  def test_experimental_get_tracing_count_function(self):
+    @def_function.function
+    def double(a):
+      return a + a
+    
+    double(tf.constant(1))
```

Comment:
This needs to be `constant_op.constant` instead of `tf.constant` - that's what causes the CI failures.

Reply:
oops, I am sorry, I will fix it immediately

Reply:
Ok, I made a new commit, I should have changed the old commit?

Reply:
No need - new commits are actually better.